apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.ts

```diff
@@ -161,4 +271,192 @@ export class PaymentAsideComponent implements OnInit {
+            transaction.remarks = item.remarks;
+            transaction.vendorUniqueName = item.uniqueName;
+            this.requestObjectTogetOTP.bankPaymentTransactions.push(transaction);
+        });
+    }
+
+    /**
+     * To cancel OTP request and count down timer
+     *
+     * @memberof PaymentAsideComponent
+     */
+    public clickedCancelOtp() {
+        this.isPayclicked = false;
+        this.timerOn = false;
+        clearTimeout(this.countDownTimerRef);
+    }
+
+    /**
+     * Timer for count down OTP validation API call
+     *
+     * @param {number} remaining   count down remaining time
+     * @returns
+     * @memberof PaymentAsideComponent
+     */
+    public startTimer(remaining: number): any {
+        let min: any = Math.floor(remaining / 60);
+        let sec: any = remaining % 60;
+        min = Number(min) < 10 ? '0' + min : Number(min);
+        sec = Number(sec) < 10 ? '0' + sec : Number(sec);
+        // document.getElementById('timerElement').innerHTML = min + ':' + sec;
+        this.timerCountDown$ = of(min + ':' + sec);
+        remaining -= 1;
+
+        if (remaining >= 0 && this.timerOn) {
+            this.countDownTimerRef = setTimeout(() => {
+                this.startTimer(remaining);
+            }, 1000);
+            return;
+        }
+
+        if (!this.timerOn) {
+
+            this.timerOn = false;
+            return;
+        }
+    }
+
+
+    public initializeNewForm() {
+        this.addAccountBulkPaymentForm = this.formBuilder.group({
+            bankType: [''],
+            urn: [''],
+            totalAmount: [''],
+            bankPaymentTransactions: this.formBuilder.array([
+                this.formBuilder.group({
+                    remarks: ['', Validators.compose([Validators.required])],
+                    amount: [''],
+                    vendorUniqueName: [''],
+                })
+            ]),
+        });
+    }
+
+
+    public addAccountTransactionsFormObject(value: any) {    // commented code because we no need GSTIN No. to add new address
+        // if (value && !value.startsWith(' ', 0)) {
+        const transactions = this.addAccountBulkPaymentForm.get('bankPaymentTransactions') as FormArray;
+        transactions.push(this.initialAccountTransactionsForm(value));
+        return;
+    }
+
+    public initialAccountTransactionsForm(val: any): FormGroup {
+        let transactionsFields = this.formBuilder.group({
+            remarks: ['', Validators.compose([Validators.required])],
+            amount: [''],
+            vendorUniqueName: [''],
+        });
+        if (val) {
+            transactionsFields.get('remarks').patchValue('');
+            transactionsFields.get('amount').patchValue(val.closingBalance.amount);
+            transactionsFields.get('vendorUniqueName').patchValue(val.uniqueName);
+        }
+        return transactionsFields;
+    }
+
+
+    public removeTransactionsDetailsForm(i: number) {
```

Comment:
Missing JS doc & return type
Avoid using single character names in method param and variable names